### DISCUSSION TITLE
Composibles without issues are populated with composibles with issues

### DISCUSSION THREAD

**Discussion by leffsu** [PatilShreyas/compose-report-to-html#47] [Q&A] [answered]
Greetings!

Looking at report, I see some bits which are marked as unstable (VMs, Uri, etc). But these widgets are in the "Composibles without issues".

Is this behavior correct?

![image](https://user-images.githubusercontent.com/30364628/226608730-9111d45d-6f31-47cd-b5f7-322ce6dd2bd5.png)

**Comment by PatilShreyas** [comment, marked as answer]
@leffsu Please refer to this doc: https://github.com/androidx/androidx/blob/androidx-main/compose/compiler/design/compiler-metrics.md

They suggest that it's a good sign if the function is skippable and restartable. In your report, you can see that both these functions are skippable as well as restartable. They mention that it would be bad if _Functions that are **restartable** but **not skippable**_.

**Comment by leffsu** [comment]
Got it, thank you!

**Comment by PatilShreyas** [comment]
If you have any questions in future, feel free to discuss them in the discussions forum: https://github.com/PatilShreyas/compose-report-to-html/discussions

It would be helpful for other visitors as well 😃.
